Macros_Load.ipynb

```diff
@@ -281,9 +281,12 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "scrolled": false
+   },
    "outputs": [],
    "source": [
+    "#Sentinal App\n",
```

Comment:
Sentinel is spelled wrong. It's spelled with an 'e'. See: https://sentinel.esa.int/web/sentinel/home